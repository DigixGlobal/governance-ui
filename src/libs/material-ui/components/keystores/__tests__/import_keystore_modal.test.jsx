import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import ImportKeystoreModal from '../import_keystore_modal';

describe('<ImportKeystoreModal />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <ImportKeystoreModal
        trigger={<div />}
        createKeystore={jest.fn}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
