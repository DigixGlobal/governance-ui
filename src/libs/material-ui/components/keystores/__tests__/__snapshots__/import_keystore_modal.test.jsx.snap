// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ImportKeystoreModal /> renders correctly 1`] = `
<EZModal
  content={[Function]}
  header="Import Keystore"
  noSubmitButton={true}
  onClose={[Function]}
  size="small"
  trigger={<div />}
/>
`;
